<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClamUI</class>
 <widget class="QMainWindow" name="ClamUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ClamUI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>821</width>
      <height>341</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>801</width>
          <height>91</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
            <widget class="QPushButton" name="settingsButton">
             <property name="minimumSize">
              <size>
               <width>64</width>
               <height>64</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>64</width>
               <height>64</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="iconSize">
              <size>
               <width>64</width>
               <height>64</height>
              </size>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
             <property name="default">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Settings</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_10">
           <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
            <widget class="QPushButton" name="whitelistButton">
             <property name="minimumSize">
              <size>
               <width>64</width>
               <height>64</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>64</width>
               <height>64</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="iconSize">
              <size>
               <width>64</width>
               <height>64</height>
              </size>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
             <property name="default">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Whitelist</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_12">
           <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
            <widget class="QPushButton" name="scheduleButton">
             <property name="minimumSize">
              <size>
               <width>64</width>
               <height>64</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>64</width>
               <height>64</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="iconSize">
              <size>
               <width>64</width>
               <height>64</height>
              </size>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
             <property name="default">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Scheduler</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_2">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>801</width>
          <height>91</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_13">
           <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
            <widget class="QPushButton" name="historyButton">
             <property name="minimumSize">
              <size>
               <width>64</width>
               <height>64</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>64</width>
               <height>64</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="iconSize">
              <size>
               <width>64</width>
               <height>64</height>
              </size>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
             <property name="default">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>History</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_14">
           <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
            <widget class="QPushButton" name="quarantineButton">
             <property name="minimumSize">
              <size>
               <width>64</width>
               <height>64</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>64</width>
               <height>64</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="iconSize">
              <size>
               <width>64</width>
               <height>64</height>
              </size>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
             <property name="default">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>Quarantine</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_15">
           <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
            <widget class="QPushButton" name="updateButton">
             <property name="minimumSize">
              <size>
               <width>64</width>
               <height>64</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>64</width>
               <height>64</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="iconSize">
              <size>
               <width>64</width>
               <height>64</height>
              </size>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
             <property name="default">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>Update</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_3">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>801</width>
          <height>91</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_17">
           <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
            <widget class="QPushButton" name="scanFileButton">
             <property name="minimumSize">
              <size>
               <width>64</width>
               <height>64</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>64</width>
               <height>64</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="iconSize">
              <size>
               <width>64</width>
               <height>64</height>
              </size>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
             <property name="default">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>Scan a file</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_20">
           <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
            <widget class="QPushButton" name="scanDirectoryButton">
             <property name="minimumSize">
              <size>
               <width>64</width>
               <height>64</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>64</width>
               <height>64</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="iconSize">
              <size>
               <width>64</width>
               <height>64</height>
              </size>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
             <property name="default">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_18">
             <property name="text">
              <string>Scan a directory</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_18">
           <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
            <widget class="QPushButton" name="scanSystemButton">
             <property name="minimumSize">
              <size>
               <width>64</width>
               <height>64</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>64</width>
               <height>64</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="iconSize">
              <size>
               <width>64</width>
               <height>64</height>
              </size>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
             <property name="default">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>Scan the whole system</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_19">
           <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
            <widget class="QPushButton" name="analysisButton">
             <property name="minimumSize">
              <size>
               <width>64</width>
               <height>64</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>64</width>
               <height>64</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="iconSize">
              <size>
               <width>64</width>
               <height>64</height>
              </size>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
             <property name="default">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_17">
             <property name="text">
              <string>Analysis</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>880</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
